content/fa/docs/concepts/overview/what-is-kubernetes.md

```diff
@@ -0,0 +1,90 @@
+---
+title: کوبرنتیز چیست?
+description: >
+  کوبرنتیز یک پلتفرم قابل انتقال، قابل گسترش و کد باز است که برای مدیریت اپلیکیشنهای مبتنی بر کانتینر استفاده می‌شود و با استفاده از این پلتفرم می‌توان خودکارسازی و پیکره‌بندری توصیقی را به سادگی انجام داد.
+content_type: مفاهیم
```

Comment:
On my phone right now so hard to check with other localizations. But shouldn’t the content_type be specified in English @sftim ?

Reply:
Yep, it's a keyword.
```suggestion
content_type: concept
```